awscli/customizations/wizard/wizards/iam/_new-role-wip.yml

```diff
@@ -107,5 +109,44 @@ plan:
         condition:
           variable: wants_config_profile
           equals: yes
-#execute:
+  preview:
+    shortname: Preview
+    description: Preview results
+    values:
+      preview_none:
+        type: template
+        value: ''
+      preview_cli_command:
+        type: template
+        value: |
+          aws iam create-role --role-name {role_name} --description {role_description} --assume-role-policy-document
+
+          aws iam attach-role-policy --role-name {role_name} --policy-arn {policy_arn}
+      preview_cfn_template:
+        type: template
+        value: |
+          Resources:
+            - {role_name}:
+                Type: "AWS::IAM::Role"
+                Properties:
+                  RoleName: {role_name}
+                  Description: {role_description}
+                  AssumeRolePolicyDocument:
+                  ManagedPolicyArns:
+                    - {policy_arn}
+      output_type:
+        type: prompt
+        description: Select an execution preview format
```

Comment:
I'm not sure if it got missed but I was hoping that we removed `execution` from this description.